### PR TITLE
yapf everything

### PR DESCRIPTION
**Pull request by llimllib**

### CODE REVIEW

File: limbo/fakeserver.py [hunk 1]
```diff
@@ -41,16 +48,11 @@ def __init__(self, server=None, users=None, events=None, bots=None, botname="tes
             "4": User("replbot", 4, "", 0),
         }
 
-        self.bots = bots if bots else {
-            "1": Bot("1", "otherbot", [], False)
-        }
-
+        self.bots = bots if bots else {"1": Bot("1", "otherbot", [], False)}
```
Comment (topher200) [inline limbo/fakeserver.py]: I'm not in love with this change. It obviously fits on one line, but I'm not sure it helps readability.

File: limbo/limbo.py [hunk 2]
```diff
@@ -23,22 +23,28 @@
...
+
 def init_log(config):
     loglevel = config.get("loglevel", logging.INFO)
-    logformat = config.get("logformat", '%(asctime)s:%(levelname)s:%(name)s:%(message)s')
+    logformat = config.get("logformat",
+                           '%(asctime)s:%(levelname)s:%(name)s:%(message)s')
     if config.get("logfile"):
-        logging.basicConfig(filename=config.get("logfile"), format=logformat, level=loglevel)
+        logging.basicConfig(
+            filename=config.get("logfile"), format=logformat, level=loglevel)
```
Comment (topher200) [inline limbo/limbo.py]: weird that it chose to put the first param on the same line as the paren in L35 but did the opposite for L39...

File: limbo/limbo.py [hunk 3]
```diff
@@ -304,22 +333,33 @@ def main(args):
...
     server.hooks = init_plugins(pluginpath, plugins_to_load)
-    event = {'type': hook, 'text': cmd, "user": "2", 'ts': time.time(), 'team': None, 'channel': 'repl_channel'}
+    event = {
+        'type': hook,
+        'text': cmd,
+        "user": "2",
+        'ts': time.time(),
+        'team': None,
+        'channel': 'repl_channel'
+    }
```
Comment (topher200) [inline limbo/limbo.py]: cool!

File: limbo/plugins/define.py [hunk 4]
```diff
@@ -15,35 +16,34 @@ def define(word):
...
 
     if len(word.split(" ")) > 1:
         return "Please only attempt to define a single word at a time!"
 
-    request_url = 'https://od-api.oxforddictionaries.com:443/api/v1/entries/{language}/{word}'.format(**info)
-    definition_url = 'https://{language}.oxforddictionaries.com/definition/{word}'.format(**info)
+    request_url = 'https://od-api.oxforddictionaries.com:443/api/v1/entries/{language}/{word}'.format(
+        **info)
+    definition_url = 'https://{language}.oxforddictionaries.com/definition/{word}'.format(
+        **info)
```
Comment (topher200) [inline limbo/plugins/define.py]: cool!

File: limbo/fakeserver.py (hunk 1 above)
Comment (llimllib) [reply limbo/fakeserver.py]: in the small, I agree with you; some of the changes definitely are readability negatives.

However I think having a standard format is a big win in general, so I'm willing to take some small negative changes for the larger overall win

**Pull request merged**